swayidle.1.scd

```diff
@@ -65,6 +68,19 @@ Sending SIGUSR1 to swayidle will immediately enter idle state.
 
 All commands are executed in a shell.
 
+# SIGNALS
+
+swayidle responds to the following signals:
+
+*SIGTERM, SIGINT*
+	Clean up timers and exit.
+
+*SIGUSR1*
+	Immediately enter idle state.
+
+*SIGUSR2*
+	Disable/enable idle timers.
```

Comment:
what does this actually mean? the man page actually doesn't mention "idle timers"

Reply:
It was intended to mean that a SIGUSR2 signal will clear all timeouts. Sending SIGUSR2 again will create them again.

The intent is to allow an easy way to stop swyidle from taking any idle actions, and then re-enable idle actions without having to start a new instance of swayidle.

It could be reworded to something like, "SIGUSR2: Disable all timeout commands. If commands have already been disabled, re-enable them."

Reply:
The typical use-case is to disable swayidle from triggering while you're watching a video...

Reply:
-1 to this use-case. Use the Sway commands to inhibit idle instead.

Reply:
I wonder what convention there is for applications like web-browser to inhibit screensavers...

Reply:
There is a [standard Wayland protocol](https://gitlab.freedesktop.org/wayland/wayland-protocols/-/tree/master/unstable/idle-inhibit).

Reply:
@emersion I looked in to using the native sway commands, but they do not cover all use cases. For example, if I want a keyboard shortcut to disable idle, I don't think the sway commands can be used for that. You currently need to kill swayidle and then restart it. 

Adding a signal means you could just do something like: `bindsym $mod+i exec killall -USR2 swayidle` to toggle idle timeouts on or off.

Reply:
SIGUSR2 has handling has been removed from this PR